hidi/matrix.py

```diff
@@ -226,6 +227,74 @@ def transform(self, M,  **kwargs):
         return self.keras_model, kwargs
```

Comment:
`__init__` functions never need return statements, in fact, I think python will just ignore the return (but will evaluate the code in the statement). This like can :fire: